dynawo/sources/Launcher/main.cpp

```diff
@@ -54,6 +54,11 @@ namespace po = boost::program_options;
 
 using boost::shared_ptr;
 
+void usage(const po::options_description& desc) {
+  cout << "Usage: dynawo jobs-file" << std::endl << std::endl;
```

Comment:
Aren't we supposed to add the jobs file name after the jobs-file option?

Reply:
in fact it is what I try to improve here and what was confusing before, this option was never really an option and if you give only the filename it works, it what boost calls positional option, I read https://www.boost.org/doc/libs/1_39_0/doc/html/program_options/tutorial.html#id2891824 to understand what was really going here

Reply:
ok good to know